app/controllers/works_controller.rb

```diff
@@ -0,0 +1,86 @@
+class WorksController < ApplicationController
+  before_action :find_work, only: [:show, :edit, :update, :destroy]
+  def index
+    @works = Work.all
+
+    @movies = Work.sort_by_category("movie")
+    @books = Work.sort_by_category("book")
+    @albums = Work.sort_by_category("album")
+    #
+    unless @works
+      # how to trigger this and make this actually occur?
+      head :not_found
```

Comment:
If there are no works, `Work.all` will return an empty array, which is truthy. Only `nil` or `false` will ever make this `unless` block happen.

However, I'm not sure `not_found` is what you want if there are no works. Wouldn't it make sense to just show 3 empty tables?